invoicing_app/management/commands/generate_tax_receipts_new.py

```diff
@@ -96,6 +94,15 @@ def __init__(self, *args, **kwargs):
 
         super(Command, self).__init__(*args, **kwargs)
 
+    def localice_date(self, country_code, date):
```

Comment:
Typo: use localize_date. Not to be a pain bugging with this, but typos make searching harder